lib/lambda_ethereum_consensus/validator/duties.ex

```diff
@@ -24,66 +25,76 @@ defmodule LambdaEthereumConsensus.Validator.Duties do
           index_in_committee: Types.uint64()
         }
 
-  @type proposer_duty :: Types.slot()
+  @type proposer_duty :: Types.validator_index()
+
+  @type sync_committee_duty :: %{
+          last_slot_broadcasted: Types.slot(),
+          subnet_ids: [Types.uint64()],
+          validator_index: Types.validator_index()
+        }
 
   @type attester_duties :: [attester_duty()]
   @type proposer_duties :: [proposer_duty()]
+  @type sync_committee_duties :: [sync_committee_duty()]
 
   @type attester_duties_per_slot :: %{Types.slot() => attester_duties()}
   @type proposer_duties_per_slot :: %{Types.slot() => proposer_duties()}
 
-  @type kind :: :proposers | :attesters
-  @type duties :: %{kind() => attester_duties_per_slot() | proposer_duties_per_slot()}
+  @type kind :: :proposers | :attesters | :sync_committees
+  @type duties :: %{
```

Comment:
We should update the `validator_set` struct to match this.

Reply:
Great catch!

Reply:
Done!